test/common/http/http2/frame_replay.h

```diff
@@ -52,10 +52,10 @@ class CodecFrameInjector {
 public:
   CodecFrameInjector(const std::string& injector_name);
 
-  void write(const Frame& frame);
+  // Writes the data using the Http::Connection's nghttp2 session.
```

Comment:
Some of the comments in this file need updating I think to reflect the new role of these classes as just static mock/environmental placeholders.